content/docs/develop/operate/upgrade/index.md

```diff
@@ -7,6 +7,8 @@ keywords: upgrade
 
 ## Upgrade from 0.6.2 to 0.7
 
+:warning: **Warning:** *Keptn 0.7 uses Helm 3.0 while previous Keptn releases rely on Helm 2.0*. By using this upgrader, **all** Helm releases are upgraded from Helm 2.0 to 3.0. This also includes Helm releases that are not managed by Keptn. If you have Helm releases on your cluster that are on version 2.0 and you do not want to upgrade, don't use this upgrader. Please take into account that the end-of-life period of Helm 2.0 begins on [August 13th, 2020](https://helm.sh/blog/covid-19-extending-helm-v2-bug-fixes/).  
```

Comment:
@grabnerandi & @christian-kreuzberger-dtx: Can I please get your feedback on that paragraph?